docs/sources/ingest-and-analyze-profile-data/pyroscope-ui.md

```diff
@@ -1,35 +1,120 @@
 ---
-title: Use Pyroscope UI
+title: Use Pyroscope UI to explore profiling data
 menuTitle: Use the Pyroscope UI
-description: How to use the Pyroscope UI to analyze performance of your applications.
+description: How to use the Pyroscope UI to explore profile data.
 weight: 40
 keywords:
   - pyroscope
   - performance analysis
   - flamegraphs
 ---
 
-## Pyroscope: Continuous Profiling in Action
-
-![Screenshots of Pyroscope's UI](https://grafana.com/static/img/pyroscope/pyroscope-ui-diff-2023-11-30.png)
+# Use the Pyroscope UI to explore profiling data
 
 Pyroscope's UI is designed to make it easy to visualize and analyze profiling data.
 There are several different modes for viewing, analyzing, uploading, and comparing profiling data.
-We will go into more detail about these modes in the [Pyroscope UI documentation].
-For now, it's important to note that one of the major benefits of continuous profiling is the ability to compare and diff profiling data from two different queries:
 
-- Comparing two different git commits before and after a code change
-- Comparing Staging vs production environments to identify differences in performance
-- Comparing performance between two different a/b tests or feature flag experiments
-- Comparing memory allocations between two different time periods before and after a memory leak
-- etc
+![Screenshots of Pyroscope's UI](https://grafana.com/static/img/pyroscope/pyroscope-ui-diff-2023-11-30.png)
+
+While code profiling has been a long-standing practice, continuous profiling represents a modern and more advanced approach to performance monitoring. This technique adds two critical dimensions to traditional profiles:
+
+Time
+: Profiling data is collected _continuously_, providing a time-centric view that allows querying performance data from any point in the past.
+
+Metadata
+: Profiles are enriched with metadata, adding contextual depth to the performance data.
+
+These dimensions, coupled with the detailed nature of performance profiles, make continuous profiling a uniquely valuable tool.
+Pyroscope's UI enhances this further by offering a convenient platform to analyze profiles and get insights that are impossible to get from using other traditional signals like logs, metrics, or tracing.
+
+In this UI reference, you'll learn how Pyroscope parallels these other modern observability tools by providing a Prometheus-like querying experience. More importantly, you'll learn how to use Pyroscope's extensive UI features for a deeper insight into your application's performance.
+
+## Key features of the Pyroscope UI
+
+The following sections describe Pyroscope UI capabilities.
+
+<!-- Add a screenshot with numbered parts for each of the sections described below. -->
```

Comment:
I think adding a screenshot with numbered sections that relate to all of the sections below would be very useful.